docs/markdown/02-k8s_origins/00-origins.md

```diff
@@ -0,0 +1,92 @@
+<!-- .slide: class="transition bg-light blue" data-background="./assets/images/k8s/transition_orchetrateur_origin.jpg"-->
+
+# Kubernetes : les origines
+
+
+##==##
+
+# Google : 15 ans de containers
+
+<br>
+<br>
+
+![center h-650](./assets/images/k8s/g3f0c37370d_0_482.png)
+
+Notes:
+Google, 15 ans d’expérience sur l’orchestration de containers
+
+Borg = c++, ~2003
+
+
+
+##==##
+
+# Cloud Native Computing Foundation
+
+
+![center h-850](./assets/images/k8s/k8s-cncf.png)
+
+https://github.com/cncf/landscape <!-- .element: class="center" -->
+
+
+Notes:
+Kubernetes réécrit en Go par les Googlers
+
+Offert à la Cloud Native Computing Foundation en 2015
+
+Kubernetes n’est qu’un outils parmi tous les outils pouvant vous aider à créer des architectures “Cloud-Natives”
+
+
+La Fondation Linux (en anglais "Linux Foundation") est un consortium à but non lucratif fondé le 21 janvier 2007
+
+Elle a pour mission de protéger et standardiser Linux en procurant les ressources et services centralisés
+
+La Linux Foundation regroupe 70 membres
+
+##==##
+<!-- .slide: class="two-column-layout with-code"-->
+
+# Kubernetes, pour quoi faire ?
+
+![float-left](./assets/images/k8s/g3f0c37370d_0_502.png)
+
+##--##
+
+<br>
+<br>
+
+<ul>
+    <li class="fragment fade-in">Lancer 5 containers basés sur l’image <code>redis:4.0</code></li>
+    <li class="fragment fade-in">Mettre en place un load-balancer interne au cluster pour servir ces 5 containers</li>
+    <li class="fragment fade-in">Lancer 10 containers <code>webapp:1.0</code></li>
+    <li class="fragment fade-in">Mettre en place un load-balancer public pour permettre d’accéder aux containers de l’extérieur du cluster</li>
+    <li class="fragment fade-in">Augmenter le nombre de containers webapp pendant les soldes 😉</li>
+    <li class="fragment fade-in">Continuer à servir les requêtes pendant la mise à jour vers <code>webapp:2.0</code></li>
+</ul>
+
+
+Notes:
+Orchestration de containers dans un cluster de machines
+
+“barreur d'un navire” en grec 
+
+##==##
+
+# Mais aussi
+
+<br>
+<br>
+
+<ul>
+    <li class="fragment fade-in">Mise à l’échelle automatique</li>
```

Comment:
si tu mets pas le fade-in, ça fait quoi ? car c'est pas l"effet par défaut ?

Reply:
En fait si, c'est pareil avec ou sans, je vais l'enlever pour simplifier du coup